build/bin/build-client-source

```diff
@@ -86,11 +86,31 @@ repack_toolkit() {
     "
 }
 
+prepare_vmagent_tarball() {
+    local repo_name=vmagent
+    local commit_hash=${VMAGENT_COMMIT_HASH:-vmagent_commit_hash}
+    local commit_short=${commit_hash::6}
+    local tarball=${source_dir}/${repo_name}-${commit_short}.tar.gz
+    local repo_var=$(
+        echo "${repo_name}_commit" \
+            | sed -e 's:-:_:g' \
+            | tr '[[:lower:]]' '[[:upper:]]'
+    )
+    echo "${repo_var}=$commit_short" >> ${client_properties}
+
+    if [ -f "${tarball}" ]; then
+        echo ${tarball} already exists, skipping build
```

Comment:
you are going to skip download not build

Reply:
Fixed it there, but it seems that "already exists, skipping build" is used incorrectly in other places too